docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx

```diff
@@ -118,21 +118,106 @@ a couple community contributors which also helped shape the following `diffusers
 - [SytanSD](https://github.com/SytanSD)
 - [bghira](https://github.com/bghira)
 - [Birch-san](https://github.com/Birch-san)
+- [AmericanPresidentJimmyCarter](https://github.com/AmericanPresidentJimmyCarter)
 
-#### 1.) Ensemble of Expert Denoisers
+#### 1.) Ensemble of Expert Denoisers (Timesteps)
```

Comment:
Can we leave this section as is since we don't use `use_karras_sigmas=True` here. It would be great if we could just convey that `denoising_start` and `denoising_end` can be specified with both `int` and `float` and state each other's advantages. I don't think we should make two full sections here.